### ISSUE TITLE
kubelet unit test flake

### ISSUE THREAD

**Issue by kargakis** [kubernetes/kubernetes#19657]
build-log [here](https://00e9e64bacdc4dc52f5c2bbcf1716bf499820d5baa5f8f6935-apidata.googleusercontent.com/download/storage/v1_internal/b/kubernetes-jenkins/o/pr-logs%2Fdd84bba64c8b44887bbe0fb0d72d4b820c062ea5%2Fkubernetes-pull-test-unit-integration%2F9301%2Fbuild-log.txt?qk=AD5uMEu81ARZy6touYt8BztowQEbf1NNJjAy07P7SCGtvGIQjo8iWfS775tc3b-S7asHoEyCbfIHEQCLSj0itZnanbfP4Jzve4UEcv0bRIrVGsIg1sM5zym5gD4MBXWjHTV5TUjvDJyDgDSnOgZ0gZafZuCy4v95CN9VjDQyS5-tv92iZ1ohGq-O08lbDxdQD66PvQv-x1ksvb4Q2p0b4H3ivW32DIA9fUz4Gmf1nwKT3OdcCLAUPbJd858aXdljMKbgYaqJlcPCBh96zBplRepy7xGyJ6DGlpJ6N7BqTsC0BLXvf61e-zLWrHUe6ov9Ptj9JgY3yFLG71pGQ2BDJo5XHeiDTg2jG-4_2QBt2i-sAkLlZBIkToqaovcPw8SMfwx3WHjfXLbFFxyX2jqYPLfaFBZR-y--272ZTJa9YkKpSaO4PsTgO29FrY-sD3EKMnsyYi1mqzfJhI6xSdUx32qEOEdIk4TAyPB_DsjaUpQlhGibpDFHrw4yxm4lPB-YlpTHYdM1zhKnr3SPZDkXeifd4YMmkqh7qSmqCmix1rzf-vUPNZMlLl5V_aZIJBzRDbmPkP-h9vENmzv-Uz-poHEhBGORzr9wg0itOSpwMls2tudps_hLbntazSR1a2mdI3lbhIP0pKH4CqAgA3gyzFteM_vii44MCcltAhUpXbQteXCBtpsc9g_vaDmx2w0pDewWwhMsg9U0yEVtuHPKkgpl1pZS36y1NnkPAMt4eQeVgj7rNMS0aCJc8K_FrkeDPRo_sZGFRf0EnUhYi4I_24449GMRVSDHAStA_B7zcMSjJMEjBXwakidOyB2PHB3lDOgBnbOn67rwbz1yq1Cq7MoGSMXofeDQwoDgrHDG3X4UWYK5vtxKwxqCGAlqxlWtz62ZrjiryP9cDhfUJ4ynd_rAHgDmBtm0PPJ-f9uuOHyPf0fqFvoiLMxlytzKERwuugZ7UCpzdmQqhnaf7Eg6GNSMOF0RrvSLaQ)

```
FAIL	  k8s.io/kubernetes/pkg/kubelet/status	2.188s
```

**Comment by yujuhong** [comment]
status.TestStaticPodStatus

 Stack Trace
```
<autogenerated>:31: 
                        
	Error Trace:	manager_test.go:488
	
	Error:		Should be true
	
	Messages:	Expected: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:30 +0000 UTC ContainerStatuses:[]}, Got: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:29 +0000 UTC ContainerStatuses:[]}
	
<autogenerated>:31: 
                        
	Error Trace:	manager_test.go:490
	
	Error:		Should be true
	
	Messages:	Expected: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:30 +0000 UTC ContainerStatuses:[]}, Got: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:29 +0000 UTC ContainerStatuses:[]}
	
<autogenerated>:31: 
                        
	Error Trace:	manager_test.go:500
	
	Error:		Should be true
	
	Messages:	Expected: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:30 +0000 UTC ContainerStatuses:[]}, Got: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:29 +0000 UTC ContainerStatuses:[]}
	
<autogenerated>:31: 
                        
	Error Trace:	manager_test.go:521
	
	Error:		Should be true
	
	Messages:	Expected: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:30 +0000 UTC ContainerStatuses:[]}, Got: {Phase: Conditions:[] Message:5440260128355399364 Reason: HostIP: PodIP: StartTime:2016-01-14 16:29:29 +0000 UTC ContainerStatuses:[]}
```

**Comment by Random-Liu** [comment]
Should be fixed in #19690.

**Comment by Random-Liu** [comment]
Close this issue because #19690 has been merged. If similar error occurs again, feel free to reopen this issue.
